### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by kmike** [scikit-learn-contrib/lightning#96]
Hey, 

What do you think about making a new release? Is there anything blocking it?

**Comment by fabianp** [comment]
I don't think there has been much activity since last release. Is there
something that would justify a new release?

**Comment by kmike** [comment]
Nothing big; I want to add lightning support to https://github.com/TeamHG-Memex/eli5 library, and it requires `classes_` attribute which is new in master. Installing from git works fine, but it is nice to point to a release.

**Comment by fabianp** [comment]
OK cool. I need to fix a couple of things and I'll make a release next week.

**Comment by fabianp** [comment]
This is done, the new package is in pypi and its also slowly hitting conda-forge

**Comment by kmike** [comment]
Thanks @fabianp!
